server/routes/testutils/testData.ts

```diff
@@ -0,0 +1,112 @@
+import { PrisonerGender } from '../../data/DraftAdjudicationResult'
+import PrisonerSimpleResult from '../../data/prisonerSimpleResult'
+import { Location } from '../../data/PrisonLocationResult'
+import { ReportedAdjudicationStatus } from '../../data/ReportedAdjudicationResult'
+
```

Comment:
Obviously this file needs developing further/refactoring so we can use it to generate data for other tests - so it's a work in progress